modules/operations/pages/monitoring/stream-audit-logs.adoc

```diff
@@ -0,0 +1,87 @@
+= Stream {astra_db} audit logs 
+
+Stream your xref:astra-db-serverless:administration:view-account-audit-log.adoc[{astra_db} audit logs] through {product_name} to an external system.
+
+To enable audit log streaming, you must provide the **Full Name** of your {product_name} topic and the streaming tenant's `client.conf` file to {support_url}[{company} Support] or your account representative, or POST your configuration to the https://docs.datastax.com/en/astra-api-docs/_attachments/devops-api/index.html#tag/Organization-Operations/operation/configureTelemetry[Astra DevOps API telemetry endpoint].
+
+== Create an {product_name} topic for audit logs
+Audit log streaming requires a streaming tenant in the AWS `us-east-2` region.
+You can create a new tenant with the xref:astra-streaming:getting-started:index.adoc[{product_name} quickstart] or use an existing {product_name} tenant.
+
+. In the {link-astra-portal}, go to **Streaming**.
+. Select an existing tenant or xref:astra-streaming:getting-started:index.adoc#your-first-streaming-tenant[Create a Streaming Tenant] in AWS `us-east-2`.
+. Add a xref:astra-streaming:getting-started:index.adoc#add-a-namespace-to-hold-topics[namespace] and xref:astra-streaming:getting-started:index.adoc#a-topic-to-organize-messages[topic] to the tenant.
+. On the *Namespace and Topics* page, click the new topic, and then copy the topic's **Full Name**, such as `persistent://aws-us-east-2-mk/*NAMESPACE_NAME*/*TOPIC_NAME*`.
+. If necessary, create additional audit log topics, and then record the **Full Name** for each topic.
+Additional topics can help you organize audit logs by event type or other criteria.
+. In the {link-astra-portal}, go to **Streaming**, and then click your audit log streaming tenant.
+. On the *Connect* tab, click **Download client.conf**.
+. Send your topic's full address and the `client.conf` file to your DataStax representative. DataStax will complete the setup.
+
+== Configure the audit log with the DevOps API
+
+You can configure audit log streaming using the Astra DevOps API instead of contacting your DataStax representative.
+
+To do this, send a POST request to the Astra DevOps API telemetry endpoint. Include your topic’s full address and values from the `client.conf` file in the request. You'll need an **Organization Administrator** token to use this API.
```

Comment:
```suggestion
Ypu can use the xref:astra-api-docs:ROOT:attachment$devops-api/index.html#tag/Organization-Operations/operation/configureTelemetry[Astra DevOps API telemetry endpoint] to configure audit log streaming instead of providing the configuration details to {company} Support.

. In the {link-astra-portal}, create an application token with the **Organization Administrator** role, if you don't already have one.
```